Makefile

```diff
@@ -38,8 +38,7 @@ lint-examples:
 	golangci-lint run -v -E ${LINT_SETTINGS}
 
 lint: | lint-examples
-	golangci-lint run --timeout 2m0s -v -E ${LINT_SETTINGS},gomnd && \
-	make check-comments;
+	golangci-lint run --timeout 2m0s -v -E ${LINT_SETTINGS},gomnd
```

Comment:
the double `context.Context` in `sequenceBlocks` causes a linting issue that can't be bypassed 😢